src/main/scala/org/zalando/spearheads/innkeeper/dao/DeletedRoutes.scala

```diff
@@ -0,0 +1,16 @@
+package org.zalando.spearheads.innkeeper.dao
+
+import java.time.LocalDateTime
+
+import org.zalando.spearheads.innkeeper.dao.MyPostgresDriver.api._
+
+object DeletedRoutes extends TableQuery(new DeletedRoutesTable(_))
+
+class DeletedRoutesTable(tag: Tag)
+    extends Table[(String, LocalDateTime)](tag, "DELETED_ROUTES") {
+
+  def name = column[String]("NAME")
+  def deletedAt = column[LocalDateTime]("DELETED_AT")
+
+  def * = (name, deletedAt)
```

Comment:
We don't need any extra information. The sole purpose of this table is so that we can notify that the route with that name has been deleted. Everything else should go to an Audit table.
This table will be regularly purged based on deletedAt. An unique id is redundant.